serverless/Dockerfile.j2

```diff
@@ -0,0 +1,21 @@
+FROM mhart/alpine-node:{{NODE_MAJOR}}
+
+ARG CI_HELPER_VERSION
+ARG PIPENV_VERSION
+ARG PIP_VERSION
+ENV PYTHON_PIP_VERSION ${PIP_VERSION}
+
+RUN echo "Starting ..." && \
+    echo "Install system dependencies for python and pip" && \
+    apk add --update -q --no-cache bash build-base curl git libffi-dev make openldap-dev openssh-client python3 py3-pip rsync tzdata zip && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
+    ln -s /usr/bin/pip3 /usr/bin/pip && \
+    pip install pip==${PYTHON_PIP_VERSION} && \
+    echo "Done base install!" && \
+    echo "Install basics Python tools" && \
+    pip install --upgrade pipenv==${PIPENV_VERSION} && \
+    echo "Done basic python tools install" && \
+    echo "Install serverless" && \
+    yarn add serverless@${SERVERLESS_VERSION} && \
```

Comment:
Shouldn't you install it globally ?

Reply:
Done, I also added serverless-python-requirements package as it should be useful when deploying python lambdas.